### PR TITLE
main.html: Add built by coala

### LINKED ISSUE #22: Add "built by coala"
This website should mention that it has been built by the coala team.

This is a design task; do just add "built by coala".

### PR DESCRIPTION
**Pull request by pshreya2181**
main.html: Add built by coala
added the built by coala in footer
fixes https://github.com/coala/gci-leaders/issues/22

### CODE REVIEW

**Review by Monal5031 — CHANGES_REQUESTED** [review]
This is your 2nd PR for the same issue, so if you close this one you'll be unassigned from the task.
Since there are 5 commits you need to squash them into one and also you need to rebase too.

**Comment by Monal5031** [comment]
@pshreya2181 First rebase your PR then do the squash, you can leave messages on zulip if no one is online, there is no need that the mentors should be online for that to discuss, maybe some other person can also help you with that.

File: main.html
Comment (Monal5031) [inline main.html]: Why did you add this new file? There is no such need I guess.

**Review by Monal5031 — CHANGES_REQUESTED** [review]

File: main.html
Comment (pshreya2181) [reply main.html]: I thought since I closed my first PR all my changes are gone. So I uploaded the main.html with my changes in the footer.

Comment (Monal5031) [reply main.html]: No please delete this.

**Comment by pshreya2181** [comment]
I didnt close my PR but it shows closed. I was only trying to squash my changes. please let me know what is needed now.

**Comment by Monal5031** [comment]
Reopen it and then we can proceed with the squashing.

**Comment by pshreya2181** [comment]
Thanks but  I dont see reopen option..will you be able to reopen?
https://github.com/coala/gci-leaders/pull/134

**Comment by Monal5031** [comment]
No I can't reopen any PR, you can reopen this PR, you'll see the re-open PR button below, beside the comment button.

**Comment by pshreya2181** [comment]
"Reopen and comment " button next to "comment" button is disabled as it looks like it will be enabled only if I had a new commits

when I issue this command to squash - git rebase -i upstream/master, it doesnt show any picks..

image

**Comment by pshreya2181** [comment]
image

**Comment by pshreya2181** [comment]
please check if my changes are available for your review. As you can see The PR was closed during the force push.

**Comment by pshreya2181** [comment]
image

Unable to reopen PR #134 (may be permissions). what is the solution ? Thanks!

**Comment by andrewda** [comment]
If a PR doesn't have any commits, GitHub automatically closes it because there's nothing to merge. In this case, your branch is even with `coala/gci-leaders:master`. Push a commit and you'll be able to reopen this. Also in the future, don't create PR's from your `master` branch.

**Comment by pshreya2181** [comment]
shreya-patch-1 is the branch in my fork.  did commit of main.html from this branch but still PR was not reopened. ( see the above gitCMD screenshot.

**Comment by andrewda** [comment]
This PR is opened from [`pshreya2181:master`](https://github.com/pshreya2181/gci-leaders/tree/master) (see: the top of this PR). Push there.

**Comment by pshreya2181** [comment]
I have commited the main.html into master. 
image

**Comment by pshreya2181** [comment]
please review and let me know if this is good to go.

File: templates/main.html
Comment (andrewda) [inline templates/main.html]: Maybe i18nize the span? Something like:

```html
<small><span data-i18n="built-by">Built by</span> <a href="https://coala.io/">coala</a>.</small>
```

**Review by andrewda — CHANGES_REQUESTED** [review]
Also squash your commits into one.

File: templates/main.html
Comment (pshreya2181) [reply templates/main.html]: i thought of adding i18n but thought since  translation strings are not available for other languages, I didnt need it. It can be done but first I want to finish this stupid checkins, where I spent 5 days when I made the actual fix  in 1 hour.

**Comment by pshreya2181** [comment]
please check if the squash/commits are good.

**Comment by andrewda** [comment]
Not sure why @danes44 is the committer in your new commit? Also make sure your commit body contains `Closes https://github.com/coala/gci-leaders/issues/22`.

**Comment by pshreya2181** [comment]
I have already have the comments  as below when I committed my changes -
main.html: Add built by coala
added the built by coala in footer
fixes #22.

**Comment by andrewda** [comment]
That's for your pull request, not your commit 😛  See: https://github.com/coala/gci-leaders/pull/134/commits/eba645e0e75fa322997ef4f15faa658da9df669e

**Comment by pshreya2181** [comment]
thanks see now.

**Comment by jayvdb** [comment]
See https://github.com/coala/gci-leaders/pull/134/commits ; click the `...` ; you commit message doesnt include the mandatory last line

```

Fixes https://github.com/coala/gci-leaders/issues/22
```

**Comment by pshreya2181** [comment]
please check now.

**Comment by pshreya2181** [comment]
I think this is it. Thanks mentors and reviewers.

**Review by andrewda — APPROVED** [review]

**Comment by andrewda** [comment]
Actually before this is merged, it would be nice to get i18n done.

**Comment by pshreya2181** [comment]
ok, I will add the changes in en.json , es.json and main.html. Do the commit message should be for each file?

**Comment by pshreya2181** [comment]
typo - no es.json .. only 2 files en.json and main.html

File: templates/main.html
Comment (andrewda) [inline templates/main.html]: Don't get rid of this ending span tag!

Comment (andrewda) [inline templates/main.html]: Don't get rid of this closing span tag!

File: templates/main.html
Comment (andrewda) [inline templates/main.html]: Add a space before the `/` in your break, like `<br />`

**Review by andrewda — CHANGES_REQUESTED** [review]
Also, you'll need to make sure to have only one commit in the PR. You can use `git commit --amend` in the future to amend onto the previous commit instead of creating a new one.

File: templates/main.html
Comment (pshreya2181) [reply templates/main.html]: issues with rebase as always.. it is picking previous commit (first) and latest commit(second) in the text editor. do I need to remove the previous commit reference (d09afc4)?

**Comment by pshreya2181** [comment]
Changes done and commited, rebased and pushed. please check.

File: templates/main.html
Comment (wisn) [inline templates/main.html:212]: `<br>` is enough.

See [here](https://stackoverflow.com/a/1946446/6914498).

**Review by wisn — CHANGES_REQUESTED** [review]

File: templates/main.html
Comment (andrewda) [reply templates/main.html:212]: We have the backslash on the line above. Maybe keep it consistent for now?

Comment (pshreya2181) [reply templates/main.html:212]: <br/>, <br /> will both work and  for consistency  I put <br /> as previous line used it. please don't make this undue  complicated. please review and close the task if it is good.

Comment (Monal5031) [reply templates/main.html:212]: You still have to merge/squash your 3 commits @pshreya2181 do it then we'll need a maintainer soon to merge this PR 😉

**Comment by pshreya2181** [comment]
done and done. please review.

**Comment by pshreya2181** [comment]
please review and close so that I can take another task.

File: templates/main.html
Comment (wisn) [reply templates/main.html:212]: @andrewda While some `hr` elements not using backslash, there is some `img` elements using backslash. We need to fix this later. Consistency is important.

**Review by wisn — APPROVED** [review]

**Comment by pshreya2181** [comment]
My task still showing - submitted for review. Please close this so that I
can take another gci task.

Thanks,
Shrerya

**Comment by pshreya2181** [comment]
hello!  please release from this task. my previous task was approved and release in an hour. I have 1 more to complete before 17th.

**Review by Monal5031 — CHANGES_REQUESTED** [review]
Sorry for late response there were things going on in college. Changes look good to me, but the commit message needs update, you have used the words `added` which should never be use, always use `add`.
Since this is not a bug fix rather a enhancement, `Closes` instead of `Fixes` should be used in commit message.

**Comment by pshreya2181** [comment]
Seriously you comment about the comments after one of your guys approved it
after a day. I regret taking this task. I see no coordination between you
guys and you guys seems to have forgotten this is a GUI beginner task.

**Comment by pshreya2181** [comment]
Please all of you agreee what should be the exact comment and paste here. I will use it. I am tired of changing every trivial thing from  each one of you.

**Comment by Monal5031** [comment]
@pshreya2181 If you'd have read the docs, clearly it is mentioned there on how to format our commit messages which is mentioned in task description, (link: [http://coala.io/commit](http://coala.io/commit)) . I mentioned the changes you need to do, reframe sentence to use `add` not `added` as we should always use imperative sentences. Also `Closes` not `Fixes` as it is not a bug fix but an enhancement.

Edit: It was already mentioned in your last PR https://github.com/coala/gci-leaders/pull/129#pullrequestreview-87065274 to do so.

**Comment by pshreya2181** [comment]
It all should have done before approving it. There should be a stop for
changes at some point. What is the guarantee that another mentor doesn’t
comment  about these comments to change to something else. So please agree
among yourselfs and provide the comments and I will update for one last
time.
Thanks

**Comment by Monal5031** [comment]
@pshreya2181 I agree that this being a small PR is taking too long and it's becoming frustrating for you but everyone has comply to docs, its usually hard at start but you will get used to it. 

Please make the changes I suggested they are according to our commit message docs.
Also if a mentor approves a PR and another mentor requests some changes it doesn't mean we don't have co-ordination between us, it must've got missed by the first mentor we are humans after all.

Again to sum up the changes I requested:
Your commit message head is correct no changes there
In body you used `added` which should not be used, rather reframe the sentence to use `add`.
Its not a bug fix so use `Closes` not `Fixes`

**Comment by pshreya2181** [comment]
Updated commit comments.. please review and let me know. thanks!

**Comment by jayvdb** [comment]
You used `closes` instead of `Closes`.

**Comment by jayvdb** [comment]
unack https://github.com/coala/gci-leaders/pull/134/commits/baae0e7b58636b48c9c9fdf05b2cd20895b59d7f

**Comment by pshreya2181** [comment]
That is the reason I asked for exact comment string as you guys are so particular about cases, spaces, commas or anything.  Updated now. please check.

**Comment by pshreya2181** [comment]
hi! what is the typical time you need to approve it and release me from
this task. appreciate if you can do this asap.
thanks

**Review by Monal5031 — APPROVED** [review]
LGTM 👍  thanks for bearing with us.

**Comment by Monal5031** [comment]
ack 8960f21

**Comment by pshreya2181** [comment]
please check. and complete the task.I really have to complete another task
before 17th. thanks!

**Comment by Monal5031** [comment]
waiting for @jayvdb to merge it.

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/134/commits/8960f214a47af27ccd31361309d8fd1d5cd9d238

**Review by blazeu — APPROVED** [review]

**Pull request merged**
